index.bs

```diff
@@ -1592,36 +1621,71 @@ Constructors</h4>
 			<span class="synchronous">When creating an {{AudioContext}},
 			execute these steps:</span>
 
-			1. Set a {{[[control thread state]]}} to <code>suspended</code> on the {{AudioContext}}.
+			1. Let |context| be a new {{AudioContext}} object.
 
-			2. Set a {{[[rendering thread state]]}} to <code>suspended</code> on the {{AudioContext}}.
+			1. Let |document| be the current settings object's
+				[=associated Document=].
 
-			3. Let <dfn attribute for="AudioContext">[[pending resume promises]]</dfn> be a
-				slot on this {{AudioContext}}, that is an initially empty ordered list of
-				promises.
+			1. Set a {{[[control thread state]]}} to <code>suspended</code> on
+				|context|.
 
-			4. If <code>contextOptions</code> is given, apply the options:
+			1. Set a {{[[rendering thread state]]}} to <code>suspended</code> on
+				|context|.
 
-				1. Set the internal latency of this {{AudioContext}}
-					according to <code>contextOptions.{{AudioContextOptions/latencyHint}}</code>, as described
-					in {{AudioContextOptions/latencyHint}}.
+			1. Let 
+				<dfn attribute for="AudioContext">[[pending resume promises]]</dfn> be
+				a slot on |context|, that is an initially empty ordered list of
+				promises.
 
-				2. If <code>contextOptions.{{AudioContextOptions/sampleRate}}</code> is specified,
-					set the {{BaseAudioContext/sampleRate}}
-					of this {{AudioContext}} to this value. Otherwise, use
-					the sample rate of the default output device. If the
-					selected sample rate differs from the sample rate of the
-					output device, this {{AudioContext}} MUST resample the
-					audio output to match the sample rate of the output device.
+			1. If <code>contextOptions</code> is given, perform the following
+				substeps:
+
+				1. If {{AudioContextOptions/sinkId}} is specified, set |sinkId| to
+					<code>contextOptions.{{AudioContextOptions/sinkId}}</code> and run
+					the following substeps:
+
+					1. If |document| is not allowed to use the feature identified by
+						<code>“speaker-selection”</code>, |sinkId| is the empty string,
+						or |sinkId| does not match any audio output device identified by
+						the result that would be provided by
+						{{MediaDevices/enumerateDevices()}}, abort these substeps.
+					
+					1. Set {{AudioContext/[[sink ID]]}} to |sinkId|.
+
+				1. Set the internal latency of |context| according to
+					<code>contextOptions.{{AudioContextOptions/latencyHint}}</code>, as
+					described in {{AudioContextOptions/latencyHint}}.
+
+				1. If <code>contextOptions.{{AudioContextOptions/sampleRate}}</code> is
+					specified, set the {{BaseAudioContext/sampleRate}} of |context| to
+					this value. Otherwise, use the sample rate of the default output
+					device. If the selected sample rate differs from the sample rate of
```

Comment:
"Otherwise, use the sample rate of the default output device." I think this sentence needs to be updated to pick the sample rate of the device selected with the `sinkId` if present.

Reply:
Yes.